src/affiliations/admin.py

```diff
@@ -45,6 +46,29 @@ class Media:
 
         css = {"all": ("css/permissions.css",)}
 
+    def get_fieldsets(self, request, obj=None):
+        """Restrict what non-superusers can view/edit."""
```

Comment:
It wouldn't hurt to add more details to this docstring. We're going a bit off the beaten path, so I think it makes sense to explain why we're doing this.

Reply:
Please let me know if the new docstring is not descriptive enough!